google-ads-examples/src/main/java/com/google/ads/googleads/examples/utils/CodeSampleHelper.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.utils;
+
+import org.joda.time.DateTime;
```

Comment:
No need to use joda after Java 8. It looks like you want [`ZonedDateTime`](https://docs.oracle.com/javase/8/docs/api/java/time/ZonedDateTime.html).

Reply:
Not sure that it makes sense to switch from `joda` here as it is already used in some code examples and I see that our current requirements list `Java 1.8+`.

Reply:
ZonedDateTime was introduced in 1.8, switched to it.